lib/tesla/middleware/retry.ex

```diff
@@ -39,12 +39,14 @@ defmodule Tesla.Middleware.Retry do
 
   ## Options
 
-  - `:delay` - The base delay in milliseconds (defaults to 50)
-  - `:max_retries` - maximum number of retries (defaults to 5)
-  - `:max_delay` - maximum delay in milliseconds (defaults to 5000)
+  - `:delay` - The base delay in milliseconds (positive integer, defaults to 50)
+  - `:max_retries` - maximum number of retries (non-negative integer, defaults to 5)
+  - `:max_delay` - maximum delay in milliseconds (positive integer, defaults to 5000)
   - `:should_retry` - function to determine if request should be retried
   """
 
+  use Bitwise, skip_operators: true
```

Comment:
Is this `use` necessary?

Reply:
Good catch, it's no longer necessary. Fixed.

Reply:
Hmm, actually we do need it, for Elixir 1.9- support. I'll add it back.

https://github.com/elixir-lang/elixir/pull/9525